chucks-pick3-mobile/src/app/components/generate-draw-time-card/generate-draw-time-card.component.ts

```diff
@@ -85,14 +91,14 @@ export class GenerateDrawTimeCardComponent implements OnInit, OnDestroy {
         this.pick3StateLottery = null;
     }
 
-    public setDrawingTimeMenuItems(selectedDrawTime, timeOfDay) {
+    public setDrawingTimeMenuItems(selectedDrawTime, timeOfDay): void {
```

Comment:
User proper method declaration.  I.E  declare parameter types.

Reply:
Resolved and refactored code.